lib/controller/StekPakketController.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+namespace CsrDelft\controller;
+
+use CsrDelft\common\CsrGebruikerException;
+use CsrDelft\entity\commissievoorkeuren\StekPakket;
+use CsrDelft\model\security\LoginModel;
+use CsrDelft\repository\StekPakketRepository;
+use CsrDelft\view\JsonResponse;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\RouterInterface;
+
+class StekPakketController {
+	private $basispakketten = [
```

Comment:
Dit is eigenlijk geen controller, maar ik neem aan dat dit zo gedaan is om het allemaal een beetje bij elkaar te houden.

Reply:
*controller logica

Reply:
Klopt, deze data zal niet buiten deze controller gebruikt worden, dus heb hem voor nu even hierin gelaten